brian2/devices/cpp_standalone/templates/synapses.cpp

```diff
@@ -10,51 +10,51 @@
 
 {% block maincode %}
 
-	// This is only needed for the _debugmsg function below	
-	{# USES_VARIABLES { _synaptic_pre } #}	
+    // This is only needed for the _debugmsg function below
+    {# USES_VARIABLES { _synaptic_pre } #}
 
-	// scalar code
-	const int _vectorisation_idx = -1;
-	{{scalar_code|autoindent}}
+    // scalar code
+    const size_t _vectorisation_idx = -1;
+    {{scalar_code|autoindent}}
 
-	{{ openmp_pragma('parallel') }}
-	{
-	std::vector<int> *_spiking_synapses = {{pathway.name}}.peek();
-	const unsigned int _num_spiking_synapses = _spiking_synapses->size();
+    {{ openmp_pragma('parallel') }}
+    {
+    std::vector<int> *_spiking_synapses = {{pathway.name}}.peek();
+    const int _num_spiking_synapses = _spiking_synapses->size();
```

Comment:
Removed unsigned for consistency, right?

Reply:
Exactly (we cannot use it in the for loop that is parallelized with OpenMP)